spec/models/comment_spec.rb

```diff
@@ -91,7 +91,8 @@
         before { comment.save(validate: false) }
 
         it "doesn't change the comment" do
-          expect(comment.update(comment_content: "Why did you block me?")).to be_falsey
+          expect { comment.update!(comment_content: "Why did you block me?") }
+            .to raise_error(ActiveRecord::RecordInvalid)
```

Comment:
Sorry, I should have mentioned on the other PR that the `.to be_falsey` pattern is OK to test for things not saving. Although IMO this is a little bit clearer in the test's intent so still a good thing to do

Reply:
Yeah, I prefer checking for the relevant error rather than falsiness -- there were two tests where the change to update! revealed we were using invalid data (e.g., random text where something that looked like an email was expected).